security/advancedtls/testdata/crl/provider/extensions.conf

```diff
@@ -0,0 +1,4 @@
+subjectKeyIdentifier = hash
+authorityKeyIdentifier = keyid,issuer
+basicConstraints = CA:FALSE
+keyUsage = digitalSignature, keyEncipherment
```

Comment:
Nit: you may want a newline at the end of this file?

Reply:
Done